spec/features/users/totp_management_spec.rb

```diff
@@ -45,8 +45,8 @@
 
       expect(
         [
-          page.find_field(t('forms.totp_setup.totp_step_1')),
-          page.find_field(t('forms.totp_setup.totp_step_4')),
+          page.find('[aria-labelledby="totp-step-1-label"]'),
+          page.find('[aria-labelledby="totp-step-4-label"]'),
```

Comment:
Is Capybara unable to find the field by its label when labelled using `aria-labelledby` ? That's a little disappointing if so, since using label finders is a nice way to have some extra guarantees that the label names are surfaced correctly to assistive technology.